nc2zarr/opener.py

```diff
@@ -91,9 +94,15 @@ def _open_mfdataset(
                 combine = 'by_coords'
                 warnings.warn(f'input/concat_dim is not specified, '
                               f'combining by coordinates')
+            if 's3' in input_paths[0]:
```

Comment:
1. This is wrong as it includes any path name with "s3" in it, for example "reflectanes32". 
2. Using the first path only is not correct, because other path may be in the local file system.

Reply:
`This is wrong as it includes any path name with "s3" in it, for example "reflectances32".`

This is a good point, I'll need to fix it first - locally.

For second part, for now, all the NetCDF files we need to convert are stored on S3 buckets. However, I will fix this contribution after completing my current iteration.